### PR TITLE
Add noscript fallback and necessary changes

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
This PR adds a barebones fallback when JavaScript is disabled telling users it's required as disused in #4149

### Preview:
Web capture_21-5-2022_1733_localhost

In this PR:
- Added `<noscript>` fallback
- Added necessary CSS to `<style>` tags since we usually load CSS with js.
- Added `lang="en"` to the ejs template since we can't set it with JS (gets overwritten by i18n when JS in enabled).
- All ejs changes was mirrored to to snowpacks `index.html` for parity.

### CODE REVIEW

File: web/views/pages/index.ejs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="http://ogp.me/ns/fb#">
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="http://ogp.me/ns/fb#" land="en">
     <head>
-        <meta http-equiv="content-type" content="text/html; charset=UTF8" />
+        <meta charset="utf-8" />
```
Comment (VIKTORVAV99) [inline web/views/pages/index.ejs:4]: This is just the short form introduced with html5, functionality is the same.

File: web/public/index.html [hunk 2]
```diff
@@ -1,44 +1,84 @@
...
+    </script>
+    <style>
+      html,
+      body {
+        margin: 0 !important;
+        padding: 0 !important;
+        font-family: 'Open Sans', sans-serif;
+        transition: background-color 800ms;
+        overflow: hidden; /* Disable dragging that can occur on mobile */
+      }
```
Comment (madsnedergaard) [inline web/public/index.html:56]: I don't think we need to implement this styling directly here, in the build it's loading CSS fine with JS disabled :)

Screenshot 2022-05-23 at 13 47 20
Screenshot 2022-05-23 at 13 47 12

Comment (VIKTORVAV99) [reply web/public/index.html:56]: Hmm it didn't when I tried it last time.

I'm not at my computer right so can't check right now but I'll test it on Edge and Firefox when I get back. (might be browser specific)

Comment (madsnedergaard) [reply web/public/index.html:56]: It makes sense if it fails during local development, but it should be the same on production no matter the browser? 🤔

Comment (VIKTORVAV99) [reply web/public/index.html:56]: Found the problem! We don't actually load the CSS bundles in html when using snowpack but rather snowpack use js to inject them (for HMR?).

I'll change back the ejs template when I get back and keep the styles in the snowpack index file when I get back.

**Comment by VIKTORVAV99** [comment]
@madsnedergaard should be all ready now!

(prettier also decided to format the entire scss file 😅)

File: web/views/pages/index.ejs [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="http://ogp.me/ns/fb#" land="en">
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="http://ogp.me/ns/fb#" lang="en">
```
Comment (madsnedergaard) [inline web/views/pages/index.ejs:2]: Good catch!

**Review by madsnedergaard — APPROVED** [review]

File: web/views/pages/index.ejs (hunk 3 above)
Comment (VIKTORVAV99) [reply web/views/pages/index.ejs]: I will attribute this one to [webhint](https://marketplace.visualstudio.com/items?itemName=webhint.vscode-webhint) 😅

**Pull request merged**